index.html

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://use.typekit.net/gjl1hek.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/font-awesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="path/to/font-awesome/css/font-awesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="path/to/font-awesome/css/font-awesome.min.css"
+    />
+
+    <script
+      src="https://kit.fontawesome.com/8c1870b6fa.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+    <title>Urban Design & Landscape Expo</title>
+  </head>
+  <body>
+    <section class="big-main">
+      <!-- header starts here -->
+      <header>
+        <a href="#home" class="logo">
+          <img src="./assests/udle-2022-main.webp" alt="logo" srcset="" />
+        </a>
+        <nav>
+          <!-- mobile navigation starts here -->
+          <a href="#home" class="logo">
+            <img src="./assests/udle-2022-main.webp" alt="logo" srcset="" />
+          </a>
+          <div class="dropdown">
+            <div id="dropdownContent" class="dropdown-content">
+              <span class="close" onclick="closeDropdown()">&times;</span>
+              <a href="#home" onclick="closeDropdown()">Home</a>
+              <a href="./about.html" onclick="closeDropdown()">About</a>
+              <a href="#cards" onclick="closeDropdown()">Program</a>
+              <a href="#cards" onclick="closeDropdown()">Join</a>
```

Comment:
- [ ] wonderful job so far. Please your design is missing the top short bar section with social media links, logout, and language links aligned to the right. kindly add that in both home and about page

![Screenshot (975)_LI](https://github.com/Kaiserabbas/Capstone/assets/65298048/415bf543-440d-4fa2-8f1e-153dfa8ead65)